src/inject/dynamic-theme/css-collection.ts

```diff
@@ -0,0 +1,89 @@
+import {styleManagers} from './index';
+import {forEach} from '../../utils/array';
+
+function beautify(text: string) {
+    const CSS = (text
+        .replace(/\{/g,'{%--%') // {
+        .replace(/\}/g,'%--%}%--%') // }
+        .replace(/\;/g,';%--%') // ;
+        .replace(/%--%\s{0,}%--%/g,'%--%') // Remove %--% Without any characters between to next %--%
+        .split('%--%'));
+    let deep = 0;
+    const formatted = [];
+    const shift = '   ';
+
+    for (let x = 0, len = CSS.length; x < len; x++) {
+        const line = CSS[x] + '\n';
+        if (line.match(/\{/)) { // {
+            formatted.push(shift.repeat(deep++) + line);
+        } else if (line.match(/\}/)) {
+            formatted.push(shift.repeat(--deep) + line);
+        } else {
+            formatted.push(shift.repeat(deep) + line);
+        }
+    }
+    return formatted.join('');
+}
+
+export function exportCSSText() {
+    const css = [];
+
+    const fallbackStyle = document.head.querySelector('.darkreader--fallback');
```

Comment:
Create a function like `addStaticCSS(selector, comment)` and do all the `querySelector, beautify, push` there.